alarm/utils.py

```diff
@@ -0,0 +1,122 @@
+from alarm.models import Coin, Candle
+import websocket
+import json
+import ssl
+
+
+def connect_binance_socket(currencies, intervals):
+    websocket.enableTrace(False)
+    socket_urls = [f'wss://stream.binance.com:9443/ws/{currency}@kline_{interval}' for currency, interval in
+                   zip(currencies, intervals)]
+
+    sockets = []
+    for socket_url in socket_urls:
+        socket = websocket.create_connection(socket_url, sslopt={'cert_reqs': ssl.CERT_NONE})
+        socket.on_message = feedback_message_from_websocket
```

Comment:
Called here and at 91. It really lacks comment why we need this 2 times.

Reply:
In the case of soket.on_message = feedback_message_from_websocket, this line of code sets the on_message method of the WebSocketApp soket instance to feedback_message_from_websocket. The default name of the function is "on_message", but Alex suggested changing the naming and we settled on "feedback_message_from_websocket". It's the same with the on_close function. And on 91 we call this function and show the logs in the terminal

Reply:
++, I agree with you